man/systemd.exec.xml

```diff
@@ -859,14 +859,17 @@
 
       <varlistentry>
         <term><varname>PAMName=</varname></term>
-        <listitem><para>Sets the PAM service name to set up a session
-        as. If set, the executed process will be registered as a PAM
-        session under the specified service name. This is only useful
-        in conjunction with the <varname>User=</varname> setting. If
-        not set, no PAM session will be opened for the executed
-        processes. See
-        <citerefentry project='man-pages'><refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum></citerefentry>
-        for details.</para></listitem>
+        <listitem><para>Sets the PAM service name to set up a session as. If set, the executed process will be
```

Comment:
when registering a PAM session you specify a "service name", which PAM then uses to find the PAM configuration file. With PAMName= you configure that service name, i.e. the service identity to use for session registration, and hence we set up a session *as* that service name.  This hence appears correct to me the way it is.